packages/rest-crud/README.md

```diff
@@ -64,7 +64,7 @@ class defined without the need for a repository or controller class file.
 
 If you would like more flexibility, e.g. if you would only like to define a
 default `CrudRest` controller or repository, you can use the two helper methods
-(`defineCrudRestController` and `defineCrudRepositoryClass`) exposed from
+(`defineCrudRestController` and `defineEntityCrudRepositoryClass`) exposed from
 `@loopback/rest-crud`. These functions will help you create controllers and
```

Comment:
IIUC the scope of the change, then this information is no longer correct, because `defineEntityCrudRepositoryClass` is no longer exported by `@loopback/rest-crud`.

BTW are you sure the benefits of removing `defineCrudRepositoryClass` from `@loopback/rest-crud` are worth the costs of publishing a semver-major version so shortly after the previous one?

Have you considered adding the following line to `packages/rest-crud/src/index.ts`?

```ts
// Perserve backwards compatibility
export {defineEntityCrudRepositoryClass as defineCrudRepositoryClass} from '@loopback/repository';
```

Reply:
Never mind, I see that `@loopback/rest-crud` has version `0.7.0` now, so breaking changes are not a problem. 

Let's make sure the documentation is correct, no need to preserve the old API.

Reply:
Since `@loopback/rest-crud` is an experimental package (0.x.y), I chose to make it a breaking change. But I'm not strongly against the backward compatibility.